tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -20,6 +20,7 @@
 <dai-copier-be.conf>
 <dai-copier-gain-module-copier-capture.conf>
 <gain-capture.conf>
+<gain-copier-capture.conf>
```

Comment:
why do you need to include this?

Reply:
As per https://github.com/thesofproject/sof/pull/6783/commits/29aa56e7ef3011b1986a5d167579c651d0c5d6c6#diff-941777fa99bc1cdc4a9e31138c87e55074cc97a429bc04bf0d9cf24d0143b55aR21  took this change but seems not mandatory for this .conf as it is not used. Will remove it.

Reply:
Done.